test/unit/algo/container/soa_vector.cpp

```diff
@@ -203,73 +560,6 @@ TTS_CASE("Check soa_vector::pop behavior")
   TTS_EQUAL(uv, uvref);
 };
 
-TTS_CASE("Check soa_vector::data behavior")
-{
-  eve::algo::soa_vector<t_t>          tv  = { t_t{1,2.3f  ,'4'}
-                                      , t_t{5,6.7f  ,'8'}
-                                      , t_t{9,10.11f,'X'}
-                                      };
-
-  eve::algo::soa_vector<udt::grid2d>  uv = { udt::grid2d{1,2}
-                                     , udt::grid2d{3,4}
-                                     , udt::grid2d{5,8}
-                                     };
-
-  auto tv_data = tv.data();
-  for(std::size_t i = 0;i< tv.size();++i)
-    TTS_EQUAL( eve::read(tv_data++), tv.get(i));
-
-  auto uv_data = uv.data();
-  for(std::size_t i = 0;i< uv.size();++i)
-    TTS_EQUAL( eve::read(uv_data++), uv.get(i));
-};
-
-TTS_CASE("Check soa_vector::begin/end behavior")
-{
-  eve::algo::soa_vector<t_t>          tv  = { t_t{1,2.3f  ,'4'}
-                                      , t_t{5,6.7f  ,'8'}
-                                      , t_t{9,10.11f,'X'}
-                                      };
-
-  eve::algo::soa_vector<udt::grid2d>  uv = { udt::grid2d{1,2}
-                                     , udt::grid2d{3,4}
-                                     , udt::grid2d{5,8}
-                                     };
-
-  auto tv_begin = tv.begin();
-  auto tv_end = tv.end();
-  int i = 0;
-  while(tv_begin != tv_end)
-    TTS_EQUAL( eve::read(tv_begin++), tv.get(i++));
-
-  auto uv_begin = uv.begin();
-  auto uv_end   = uv.end();
-  i = 0;
-  while(uv_begin != uv_end)
-    TTS_EQUAL( eve::read(uv_begin++), uv.get(i++));
-};
```

Comment:
Hey, this seems likee a good test!

Reply:
It has been moved l 145 to be next to the begin/end test. See above.